docs/src/languages/php/_index.md

```diff
@@ -22,212 +30,110 @@ Note that from PHP 7.1, the images use the Zend Thread Safe (ZTS) version of PHP
-That means functions and classes in those files are always available and don't need to be autoloaded,
-at the cost of any changes to those files requiring a PHP-FPM restart.
-Since PHP-FPM restarts on each new deploy, this feature is a major win on Platform.sh and we recommend using it aggressively.
+To have more control, you can [define rules to specify which static files you want to allow](../../create-apps/web/php-basic.md#set-different-rules-for-specific-locations).
+See also a step-by-step explanation on [how to create a basic PHP app with a front controller](../../create-apps/web/php-basic.md).
 
-To enable preloading, add a `php.ini` value that specifies a preload script.
-Any [`php.ini` mechanism](/languages/php/ini.md) works,
-but using a variable in `.platform.app.yaml` is the recommended approach:
-
-```yaml
-variables:
-    php:
-        opcache.preload: 'preload.php'
-```
+### Complete example
 
-The `opcache.preload` value is evaluated as a file path relative your [app configuration](../../create-apps/_index.md).
-It may be any PHP script that calls `opcache_compile_file()`.
+A complete basic app configuration looks like the following:
 
-The following example preloads all `.php` files anywhere in the `vendor` directory:
+```yaml {location=".platform.app.yaml"}
+name: 'app'
 
-```php
-<?php
-$directory = new RecursiveDirectoryIterator(getenv('PLATFORM_APP_DIR') . '/vendor');
-$iterator = new RecursiveIteratorIterator($directory);
-$regex = new RegexIterator($iterator, '/^.+\.php$/i', RecursiveRegexIterator::GET_MATCH);
+type: 'php:8.1'
 
-foreach ($regex as $key => $file) {
-    // This is the important part!
-    opcache_compile_file($file[0]);
-}
+web:
+    locations:
+        '/':
+            root: 'public'
+            passthru: '/app.php'
 ```
 
-{{< note >}}
-
-Preloading all `.php` files may not be optimal for your application and may even introduce errors.
-Your application framework may provide recommendations or a pre-made preload script to use instead.
-You have to determine the optimal preloading strategy for your situation.
-
-{{< /note >}}
-
-#### Preloading and dependencies
-
-Your preload script runs each time PHP-FPM restarts, including during your build.
-This means it runs before your dependencies have been installed (such as with Composer).
-
-If your preload script uses `require` for dependencies, it fails during the build
-because the dependencies aren't yet present.
-
-To resolve this, you have two options:
+## Dependencies
 
-* Have your script `include` dependencies instead of `require`
-  and fail gracefully if the dependencies aren't there.
-* Enable preloading with a variable that [isn't available during the build](../../development/variables/set-variables.md#variable-options).
-  Then preloading happens only on deploy.
+By default, to manage dependencies, PHP images use the latest Composer 1.x release as [build flavor](../../create-apps/app-reference.md#build).
 
```

Comment:
:hammer_and_wrench: This suggestion is for this and the next time together, because they should be together and don't make as much sense separately. But I can't add suggestions over deleted lines, so they're presented separately in GitHub. I modelled it on the Node.js version.

```suggestion
By default, PHP images assume you're using Composer 1.x to manage dependencies.
If you have a `composer.json` file in your code, the default [build flavor is run](https://docs.platform.sh/create-apps/app-reference.html#build):

```bash
composer --no-ansi --no-interaction install --no-progress --prefer-dist --optimize-autoloader
```
```